### PR TITLE
New package: BaileyBrown.cppcryptfs version 1.4.4.2

### LINKED ISSUE #137818: [Package Request]: cppcryptfs
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url:https://github.com/bailey27/cppcryptfs/releases
Publisher:bailey27
Package Name: cppcryptfs
Description: cppcryptfs is an implementation of the gocryptfs encrypted overlay filesystem in C++ for Windows.
Package Version: 1.4.4.2
Installer URL: https://github.com/bailey27/cppcryptfs/releases/download/1.4.4.2/cppcryptfs.exe

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

- Resolve: #137818

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/b/BaileyBrown/cppcryptfs/1.4.4.2/BaileyBrown.cppcryptfs.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: BaileyBrown.cppcryptfs
+PackageVersion: 1.4.4.2
+InstallerType: portable
+Installers:
+- InstallerUrl: https://github.com/bailey27/cppcryptfs/releases/download/1.4.4.2/cppcryptfs.exe
+  Architecture: x64
+  InstallerSha256: 2030224F3ABD45A76C4B325F67C7750E8510EF02A968C1C2F90645A2681FCADC
+- InstallerUrl: https://github.com/bailey27/cppcryptfs/releases/download/1.4.4.2/cppcryptfs32.exe
+  Architecture: x86
+  InstallerSha256: 9D6C373EFE511B51B0EF244525C5ADD6782469E936B43DE142D296BD4D5F862D
```
Comment (Dragon1573) [inline manifests/b/BaileyBrown/cppcryptfs/1.4.4.2/BaileyBrown.cppcryptfs.installer.yaml:13]:
```suggestion
  InstallerSha256: 9D6C373EFE511B51B0EF244525C5ADD6782469E936B43DE142D296BD4D5F862D
Dependencies:
  PackageDependencies:
  - PackageIdentifier: dokan-dev.Dokany
    MinimumVersion: 2.0.0.2000
```

> [!WARNING]
>
> @Exorcism0666 - Please refer to the [release details](https://github.com/bailey27/cppcryptfs/releases/tag/1.4.4.2) of the package. This version of the package requires `dokan-dev.Dokany` version 2.0. It should be added as a `PackageDependency`.
>
> @stephengillie - Latest version (currently v2.1.0.1000) of `dokan-dev.Dokany` have NOT been tested with `cppcryptfs`, users will get warings when launching the executable. I suggest adding `MaximumVersion` property, or support more flexible version-restriction syntax (such as `>=2.0,<2.1`)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 702.)

File: manifests/b/BaileyBrown/cppcryptfs/1.4.4.2/BaileyBrown.cppcryptfs.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: dokan-dev.Dokany
+    MinimumVersion: 2.0.0.2000
```
Comment (Trenly) [inline manifests/b/BaileyBrown/cppcryptfs/1.4.4.2/BaileyBrown.cppcryptfs.installer.yaml:17]: image

* Blocked by https://github.com/microsoft/winget-cli/issues/3017

**Comment by Trenly** [comment]
[Policy] Blocking Issue

File: manifests/b/BaileyBrown/cppcryptfs/1.4.4.2/BaileyBrown.cppcryptfs.installer.yaml [hunk 3]
```diff
@@ -14,6 +14,6 @@ Installers:
 Dependencies:
   PackageDependencies:
   - PackageIdentifier: dokan-dev.Dokany
-    MinimumVersion: 2.0.0.2000
+    MinimumVersion: 2.1.0.1000
```
[the remark this answers is not in the document]
Comment (Exorcism0666) [reply manifests/b/BaileyBrown/cppcryptfs/1.4.4.2/BaileyBrown.cppcryptfs.installer.yaml:17]: The fact that it is written in capital letters makes me think that you take me for an idiot. it's quite hurtful.

[the remark this answers is not in the document]
Comment (Dragon1573) [reply manifests/b/BaileyBrown/cppcryptfs/1.4.4.2/BaileyBrown.cppcryptfs.installer.yaml:17]: Sorry for that hurt 😞

According to https://github.com/microsoft/winget-pkgs/pull/137821#discussion_r1478765958, I think https://github.com/microsoft/winget-pkgs/pull/137821/commits/e91d66d06426177f74fc800d2921cd82ab2951ea might be created by accident.

> I also did these kind of things and got complaints from other developers. I can get you.

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/pull/137821#discussion_r1478765958: 

> image
> 
> * Blocked by [Add MaximumVersion to PackageDependencies winget-cli#3017](https://github.com/microsoft/winget-cli/issues/3017)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Does the "Reset-Labels" Policy command also remove https://github.com/microsoft/winget-pkgs/labels/Blocking-Issue?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**